components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -6297,7 +6315,12 @@ definitions:
           type: string
         example: [admin]
       businessInformation:
-        $ref: '#/definitions/APIBusinessInformation'
+        allOf:
+          - $ref: '#/definitions/APIBusinessInformation'
+          # - type:'#/definitions/APIBusinessInformation'
```

Comment:
lets remove the commented code

Reply:
Fixed with d8809c94c6e472619c343830c3b139c7075681ea